runners/spark/src/test/java/org/apache/beam/runners/spark/io/NumShardsTest.java

```diff
@@ -76,8 +75,7 @@ public void testText() throws Exception {
     options.setRunner(SparkRunner.class);
     Pipeline p = Pipeline.create(options);
     PCollection<String> inputWords = p.apply(Create.of(WORDS).withCoder(StringUtf8Coder.of()));
-    PCollection<String> output = inputWords.apply(new WordCount.CountWords())
-        .apply(MapElements.via(new WordCount.FormatAsTextFn()));
+    PCollection<String> output = inputWords.apply(new SimpleWordCountTest.CountWords());
```

Comment:
I don't think we need to execute `WordCount` to test `withNumShards`. Is there a `RunnableOnService` test method in `WriteTest` that might exercise this? If not, there should be. It may be just labeled `NeedsRunner` today, so it might need re-labeling.